CHANGES.md

```diff
@@ -1,3 +1,28 @@
+## edge-19.6.3
+
+* CLI
+  * Added a check for Pod Security Policy (PSP) permissions to `linkerd check`
+  * Changed `linkerd check` to only warn (not error) if PSPs are found, but
+    the caller does not have the `NET_ADMIN` capability
```

Comment:
This isn't very clear to me... Can we summarize these two points as:

* Added `PodSecurityPolicy` warnings to `linkerd check`.

Is that an accurate summary of the change?

Reply:
The task was to update `linkerd check` to ensure the caller has the RBAC permissions to create the control plane's PSP.

Reply:
The 2nd bullet point is ok, but it isn't _the caller does not have_, it's _the existing PSPs do not have_. Thanks for calling it out. I missed that.

Reply:
Perhaps:

* Updated `linkerd check` to validate the caller can create `PodSecurityPolicy` resources.

Reply:
I'll consolidate these two bullet points to the above. Thanks for the clarification